4 - AnimalShelter/Code/Animal.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AnimalShelterCode
```

Comment:
You created folder spaces (Tests & Code) however your namespace does not represent this. In most cases, namespaces *should* share the same path as the folders. So for this file and the rest you have created, the name space(s) should be either:

AnimalShelter.Code OR AnimalShelter.Tests